src/renderer/i18n/locales/en-US.js

```diff
@@ -502,7 +502,8 @@ export default {
 
     PLUGINS: {
       HEADER: 'Plugins',
-      DISCOVER: 'Discover more plugins'
+      DISCOVER: 'Discover plugins',
```

Comment:
Discover Plugins <- both words capitalized to make it consistent with other items in wallet like Create Wallet etc...